### ISSUE TITLE
Combine "selector" and "element" selectors

### ISSUE THREAD

**Issue by ottiker** [adioo/view#8]

**Comment by IonicaBizau** [comment]
What is this issue about? @adioo @gabipetrovay @lcristianiim

**Comment by lcristianiim** [comment]
I thing it was something about not working the "element" selector.

**Comment by IonicaBizau** [comment]
Ah, or maybe it's about using the `selector` and `element` fields together. :bulb:

**Comment by gabipetrovay** [comment]
Probably because they do a similar thing and having two names for the same thing would be redundant?

**Comment by lcristianiim** [comment]
Maybe it can also have positive influence. For example:
We are using classes and id's for UI, but also for logic.
When we modify the UI layout we will reorganize classes, id's and it is easy to mess up the logic that is based upon classes and id's.
Maybe it would be nice to use classes and id's only for UI, and data-element for logic.

**Comment by IonicaBizau** [comment]
@gabipetrovay No, no. `element` is now converted internally in `selector = "[data-element='<element>']"`. They are not redundant. I guess @adioo means to use it something like this:

``` sh
{
   "element": "my-element",
   "selector": ".my-class",
   "on": "click",
   "flow": "event"
}
```

...which will add `c lick` listeners to the following DOM elements (according to #7):

``` html
<div class="my-class"></div>
<span class="my-class"></span>
<span data-element="my-element"></span>
<button data-element="my-element"></button>
```
